aui.image/src/AUI/Image/webp/WebpImageLoader.cpp

```diff
@@ -71,6 +71,65 @@ _<AImage> WebpImageLoader::getRasterImage(AByteBufferView buffer) {
 
     return nullptr;
 }
+void WebpImageLoader::save(aui::no_escape<IOutputStream> outputStream, AImageView image, std::shared_ptr<WebPConfig> config, bool bUseRGBA) {
```

Comment:
There's no need to accept `std::shared_ptr<WebPConfig> config` as shared pointer. Use const reference instead: `const WebPConfig& config`